types/mixpanel-browser/index.d.ts

```diff
@@ -49,6 +49,10 @@ export interface OutTrackingOptions extends ClearOptOutInOutOptions {
     delete_user: boolean;
 }
 
+export interface RegisterOptions {
+    persistent: boolean;
+}
```

Comment:
```js
    var REGISTER_DEFAULTS = {
        'persistent': true
    };
    /**
     * Helper to parse options param for register methods, maintaining
     * legacy support for plain "days" param instead of options object
     * @param {Number|Object} [days_or_options] 'days' option (Number), or Options object for register methods
     * @returns {Object} options object
     */
    var options_for_register = function(days_or_options) {
        var options;
        if (_.isObject(days_or_options)) {
            options = days_or_options;
        } else if (!_.isUndefined(days_or_options)) {
            options = {'days': days_or_options};
        } else {
            options = {};
        }
        return _.extend({}, REGISTER_DEFAULTS, options);
    };
```

At the first, `persistent` seems not required property. And it seems `days` property, `number`-typed, should be supported 🤔 

```suggestion
export interface RegisterOptions {
    days?: number,
    persistent?: boolean;
}
```

And I can't understand why `days_or_options.days`' type is `boolean` 🤔 

```
* @param {boolean} [days_or_options.days] - number of days since the user's last visit to store the super properties (only valid for persisted props)
```

Maybe we should ask the maintainers about it.